Several changes:

GerMalaz opened the pull request:
* Added black text (needed black text on red default [Modified Puppet over 2NT]).
  * Added texdate (like iso8601 format).
  * Moved geometry to acbl2022cc_2pages [with a 5mm top margin]
  * Moved to small font size.
  * Added 4 booleans 2[dhsn]oth, defined based on strings contents(*) (and their corresponding red checkboxes)
  * Killed the page number at the bottom.
  * Changed yscale to 0.125 to fill the page (caused the pdf to have 3 pages, with an empty page 1)
  * Rearranged the 1NT section (text, checkboxes, alignment, guidelines) WIP.
  * Added footnotes for file, name, made with in both pages.
  
  (*) They defined as true if the corresponding string is not empty.
  Made it work, but probably there is a better way to do it. 
  Sorry for commiting all together.

GerMalaz committed:
Several changes:

  * Added black text (needed black text on red default [Modified Puppet over 2NT]).
  * Added texdate (like iso8601 format).
  * Moved geometry to acbl2022cc_2pages [with a 5mm top margin]
  * Moved to small font size.
  * Added 4 booleans 2[dhsn]oth, defined based on strings contents(*)
      (and their corresponding red checkboxes)
  * Killed the page number at the bottom.
  * Changed yscale to 0.125 to fill the page (caused the pdf to have 3 pages, with an empty page 1)
  * Rearranged the 1NT section (text, checkboxes, alignment, guidelines) WIP.
  * Added footnotes for file, name, made with in both pages.

GerMalaz committed:
Two pages version of kill the semicolon

mycroftw commented:
Looks like fun.  I meant this one as an example (you can tell from the - basic? - nature of the conversion) and not something I was going to straight support.

However!

I want to try this and see where it is; and I certainly will look at the changes and comment on them.  If this becomes useful, and not just a "here's what can be done", I have always thought it is an excellent idea in principle.

But note: things have been a little busy this past month, and I'm in recovery mode a bit (oh, and working on that other project, you can see over on the right.  : -).  So it might be a while before I can look at this clearly.  

That's my fault, not a statement on your work.  Thanks again!

mycroftw commented on template/acbl2022cc_2pages.sty:
```diff
@@ -44,20 +47,25 @@
 \newcommand{\tRegtext}[1]{\scriptsize \textsf{\textls[-70]{#1}}}
 \newcommand{\tRedreg}[1]{\scriptsize \textsf{\textls[-70]{\textcolor{red}{#1}}}}
 \newcommand{\tBluereg}[1]{\scriptsize \textsf{\textls[-70]{\textcolor{blue}{#1}}}}
+\newcommand{\tBlackreg}[1]{\scriptsize \textsf{\textls[-70]{\textcolor{black}{#1}}}}
```
Interesting.  I see what you're doing here, but I've always been able to "de-Alertify" with RegText.

Ah, but this is header.  Let me see how it goes.

GerMalaz replied:
Yes. My case in point here was to add "**Modified** Puppet" to twonoothertext. 
With RegText, it was shown in red. 
So I added similar commands for user text, and use `\blackbold{Modified} \blackreg{Puppet}` (pesky 5-4 in majors, or worse, 5=4) so it shows in black regardless of the default.

mycroftw commented on template/acbl2022cc_2pages.sty:
```diff
@@ -127,13 +135,15 @@
 \newboolean{smole}\newboolean{leben}\newboolean{ntneg}\newboolean{ntpen}
 \newboolean{tex4c}\newboolean{tex4d}\newboolean{tex4h}
+% private
+\newboolean{2coth}\newboolean{2doth}\newboolean{2hoth}\newboolean{2soth}\newboolean{2noth}
```
Ah, this doesn't actually change the template.  But still.

GerMalaz replied:
In general, trying to add checkboxes for these Other cases, then set them up if a string is defined, everywhere.
I see how this is a bigger deviation than just presentation.
Specially if the official, one page version lacks the space for such checkboxes.
Also, I am interested in keeping the differences to a minimum.

So, I am interested in hearing your opinion about these checkbox/text combos.
And also, if the way I did the setting of the private booleans is good, or if it is there a better way.

mycroftw replied:
I can see it - and I can see why they didn't put it in there for a one-pager.

I don't really think they add anything - the text kind of says "other" as it is - but with the wider spaces on two pages, it might get lost.  I'll look.

I don't think it fits my "change the page" level of concern - especially if it can be macro'd to be optional.  At least as long as it doesn't require a template change (which, of course, "make optional" kind of requires...  guess I have to push on that "upgrade script" a little harder)

GerMalaz replied:
Note that https://github.com/mycroftw/conv-cards/pull/91 proposes quite big template changes, and I have something bigger in mind, will explain it there.

GerMalaz commented:
Thanks! So far, had been just realigning things, redo guidelines, etc.

Didn't touch the template except for:
* Moved geometry to the acbl2022cc_2pages package  
* Defined \usertext as \small to match bigger CC size. (this should probably be moved to the  acbl2022cc_2pages too).

GerMalaz committed:
Several more changes

* Definition of Colortext/Colorbold[2] (text, color)
* Definition of colortext/colorbold[2] (text, color)
* Definition of ltrcheckbox/rtlcheckbox[4] (boolean, label, position, color)
* Definition of new private boolean footh
* Realignments of several fields and guidelines.

* Removal of geometry, define \usertextsize as \small in latex_cc_example2022.tex too.

GerMalaz committed:
Removal of original code replaced by ltrcheckbox/rtlcheckbox

GerMalaz committed:
{ltr,rtl}block

GerMalaz committed:
Many changes, WIP

GerMalaz committed:
Several changes

mycroftw commented:
Well, given the several people on https://bridgewinners.com/article/view/large-print-convention-cards/ who have come up with the same idea - and the many who say it's too tight and complicated to read - and my first comment to the CC people (that they "obviously decided up front that it had to be one-sided and fit in the convention holder", and "that that was a mistake") - looks like there's real support for this idea!  When I get out from under July, I'd better work on this one.

GerMalaz committed:
acblcc_2pages: Unify ltr/rtl functions.

Create labeledCheckbox and block.
Add a direction parameter (left/right).
Replace corresponding ltr*/rtl* functions.

GerMalaz commented:
Redone in #96, reworked, better organized.
